Scenario scenario
Vehicle vehicle
GPSCoord referenceGPS
---
